src/base/orgs/Org.ts

```diff
@@ -31,14 +31,17 @@ const GetOrgs = `
   }
 `;
 
-const GetOrgsByMembers = `
-  query GetOrgsByOwner($members: [String!]!) {
-    orgs(where: { members: $members }) {
+const GetOrgsByMember = `
+  query GetOrgsByMember($owners: [String!]!) {
+    safes(where: { owners_contains: $owners }) {
```

Comment:
This is checking whether an array contains another array. Are you sure this is what you want? Shouldn't we query for a single member here?

I'm also wondering why we don't use `members_contains` instead of going through the safe?

Reply:
We are checking currently if an array is a set of another array, since `_contains` does not accept strings.